bittide-instances/exe/post-vex-riscv-tcp-test/Main.hs

```diff
@@ -48,15 +67,15 @@ case_testGdbProgram :: Assertion
 case_testGdbProgram = do
   startOpenOcdPath <- getOpenOcdStartPath
   startPicocomPath <- getPicocomStartPath
-  startTcpPrayhPath <- getTcpSprayPath
   uartDev <- getUartDev
   gdbScriptPath <- getGdbScriptPath
 
+  putStrLn "Starting TCP Server"
+  (serverSock, _) <- startServer
   putStrLn "Starting Gdb"
```

Comment:
Do you start GDB here, or do you start it in line 130? You now print it twice.

Reply:
Fixed in #613